app/services/navigation-menu.js

```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+
+const { service } = Ember.inject;
+
+export default Ember.Service.extend({
+  onboarding: service(),
+
+  defaultMenuType: 'default',
+  onboardingMenuType: 'onboarding',
+
+  menuType: Ember.computed('onboarding.isOnboarding', function() {
+    let isOnboarding = this.get('onboarding.isOnboarding');
+    if (isOnboarding) {
+      return this.get('onboardingMenuType');
+    } else {
+      return this.get('defaultMenuType');
+    }
+  }),
+
+  isDefault: Ember.computed('menuType', 'defaultMenuType', function() {
+    return this.get('menuType') === this.get('defaultMenuType');
+  }),
+  isOnboarding: Ember.computed('menuType', 'onboardingMenuType', function() {
+    return this.get('menuType') === this.get('onboardingMenuType');
+  }),
```

Comment:
You could do 

```Javascript
Ember.computed.equal('menuType', 'default')
Ember.computed.equal('menuType', 'onboarding')
```

But then you lose the option of storing "default" and "onboarding" as property.